server/middleware/userValidator.js

```diff
@@ -107,6 +107,27 @@ class ValidateUser {
         return next();
       }).catch();
   }
+
+  /**
+   * validate userId
+   * @param {Object} request
+   * @param {Object} response
+   *
+   * @callback {Function} next
+   *
+   * @return {Object} json
+   */
+  static validateUserId(request, response, next) {
+    if (!ValidationHelper.checkValidId(request, request.params.userId)) {
+      console.log('here+++++++++++++++++++++++++++++++++++++++++++++');
```

Comment:
#### Nitpick
There should be no instance of `console.log()` in your code.

Reply:
Thank you! missed that one

Reply:
Changes have been effected